docs/audits/loads-gpt-from-official-source.md

```diff
@@ -1,24 +1,24 @@
-# Load GPT from recommended host
+# Load GPT from an official source
 
 ## Overview
 
 This audit ensures that the Google Publisher Tag (GPT) library is being loaded
-from the recommended host. The browser must open a connection to each new host
+from an official source. The browser must open a connection to each new host
 your page references, which can take significant time. By loading GPT from the
 recommended host you ensure that all ad serving requests are made to a single
 domain, reducing the number of connections to open and improving page load
 performance.
 
 ## Recommendations
 
-Ensure that GPT is being loaded from an official source. For example, standard
-integrations could use
+Ensure that GPT is being loaded from an official source. Standard integrations
+should use:
 
 ```HTML
 <script async src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"></script>
 ```
 
-And integrations with limited ads could use:
+Integrations with limited ads should use:
```

Comment:
Make "limited ads" a link to https://support.google.com/admanager/answer/9882911